spec/lib/middleware/websocket/controller_spec.rb

```diff
@@ -69,4 +69,19 @@
       expect(subject.first).to eq(controller.clients[1])
     end
   end
+
+  describe '#update_all_clients' do
+    subject { controller.update_all_clients }
+    #let(:obj) { double(:write) }
+
+    it 'calls the write method on each client' do
+      controller.on_open(incoming_client_1)
+      controller.on_open(incoming_client_2)
+      payload = controller.build_payload
```

Comment:
What's the rationale in using our own value for this? We're stubbing the code under test here (which we generally want to avoid).

Reply:
I see what you mean. I'm going to move into the next PR after this and work on my specs to try and stub less, or at least only stub the things that are out of my control. For now this test won't bother with testing with a payload, we just want to make sure that the `update_all_clients` method hits the `:write` method at the moment.